### DISCUSSION TITLE
PIP Back to Tab: Can it also pull focus on window to bring it up?

### DISCUSSION THREAD

**Discussion by crisjolliff** [FreeTubeApp/FreeTube#4648] [General]
I'd love to see the "Back to Tab" button on an inset window work a little better. If it pulled the tab to the forefront as well, that would be a great addition for my use, since I usually want to pick out another video before continuing with whatever else I'm doing.

**Comment by absidue** [comment]
We unfortunately have no control over the Picture-in-Picture window (only opening and closing it via code from the original window), so we can't change anything about it, not even the behaviour of that button (which is also why it says tab instead of window, because that's the text that chromium uses).
